grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -1592,15 +1592,17 @@ class QueryService implements Status {
     }
 
     private String getDURWhereClause(PaginationParams pagination) {
-        String where = ''
+        String where
         if (pagination.searchValue) {
-            where = ' where consent_group_key LIKE :term OR vault_export_date LIKE :term '
+            where = ' where (du.consent_group_key LIKE :term OR du.vault_export_date LIKE :term) and i.deleted = 0 '
+        } else {
+            where = ' where i.deleted = 0 '
         }
         where
     }
 
     List<ConsentCollectionLink> findCollectionLinks() {
-        String query = 'select * from consent_collection_link where deleted = 0 '
+        String query = 'select ccl.* from consent_collection_link ccl inner join issue i on i.project_key = ccl.consent_key where ccl.deleted = 0 and i.deleted = 0'
```

Comment:
The inner join, `inner join issue i on i.project_key = ccl.consent_key` restricts the CCLs to those that are only associated to undeleted consent groups. If a CCL is associated to an undeleted project, that CCL will not be captured in this query.

Reply:
Right.
If not we can return all the information. 
Currently for each consent group and project we display a link.
If a consent or project was deleted, we can remove the linkto prevent  user from navigation to this pages in Consent Collection Links table. 
What do you think?

Reply:
I am not clear about what you are suggesting.

Reply:
Hi Greg, I mean leave the query as it was and in the UI hide the consent value, if it is deleted, but displaying project.

Reply:
This feedback is about the basic functionality of the method. We have to write our underlying query methods so they return correct information. The query, as is, will leave out some qualifying rows. Whatever we do in the UI is completely irrelevant to what this function does. If I were writing this, it would cover all cases regardless of the intended use. 

Something like this (caveat, untested sql here):
```
String query = 'select ccl.* from consent_collection_link ccl 
    inner join issue i on i.project_key = ccl.consent_key or i.project_key = ccl.project_key
    where ccl.deleted = 0 
    and i.deleted = 0' 
```